specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/examples/OperationSchema_Register.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "group-name": "sample-sr-group",
+    "schema-name": "sample-schema-name",
+    "schema-content": "{\"type\":\"record\",\"name\":\"AvroUser\",\"namespace\":\"com.azure.schemaregistry.samples\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favoriteNumber\",\"type\":\"int\"}]}",
+    "serialization-type": "avro"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Location": "https://sample-namespace.servicebus.windows.net/$schemagroups/sample-sr-group/schemas/sample-schema-name/versions/3?api-version=2017-04",
+        "X-Serialization": "avro",
+        "X-Schema-Id": "a489dc4916744e3db72dda8f9693f342",
+        "X-Schema-Id-Location": "https://sample-namespace.servicebus.windows.net/$schemagroups?api-version=2017-04&schema-id=a489dc4916744e3db72dda8f9693f342",
+        "X-Schema-Version": 3
+      },
+      "body": "{\"type\":\"record\",\"name\":\"AvroUser\",\"namespace\":\"com.azure.schemaregistry.samples\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favoriteNumber\",\"type\":\"int\"}]}"
```

Comment:
This no longer matches the response schema, missing "id"

Reply:
Yeah missed that one, thanks.